tests/starwars/test_dsl.py

```diff
@@ -122,3 +122,56 @@ def test_hero_name_query_result(ds):
         }
     }
     assert result == expected
+
+
+# TODO: Keep the function name or switch to schema query names?
```

Comment:
As most names on `test_dsl.py` (I would rather say the same for `test_query.py`) make use of the query name for the test function, it is harder to guess which one is testing a specific functionality.

In this case, I wasn't sure about following the same convention and went with specifying the function that is being tested.

I do feel like it is a better approach but added the `TODO` notice to get some feedback about the naming convention for the test suite.

Reply:
Ah, ok. Keep it like that, I like how you named the test.

Reply:
Btw, since you're at it already: The `test_invalid_field_on_type_query` has a bad indentaton (the assert should not be in the `raises` block and bad spelling "doesnt".

Reply:
Done! I will push the changes after adding the last dsl test.